src/yardstick/comparison.py

```diff
@@ -147,9 +154,9 @@ class LabelComparisonSummary:
     indeterminate: int = field(init=False)
     indeterminate_percent: float = field(init=False)
 
-    false_positives: float = field(init=False)
-    true_positives: float = field(init=False)
-    false_negatives: float = field(init=False)
+    false_positives: int = field(init=False)
```

Comment:
This seems like a change in the semantics, not just a linter change. Can you help me understand why this is needed?

Reply:
mypy correctly errored out on a downstream use of these when using them to calculate another stat... I think this started as a copy-paste problem from the percentage section (which should be `float`). These stats (fp, tp, tn) are always counts of image label entries (which is never partial... always a whole number) so should be considered as `int` not `float`.